cypress/integration/RichTextEditor.spec.ts

```diff
@@ -315,6 +341,46 @@ describe('Rich Text Editor', () => {
 
           getDropdownToolbarButton().should('have.text', label);
         });
+
+        if (type !== BLOCKS.PARAGRAPH) {
```

Comment:
Considering Cypress tests are slow (I didn't check how slow though in this case) would it make sense to just run this for one heading type or ideally try all of them in one test? A `TODO` would be sufficient if you agree, I don't think we've got time for this right now.